.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+
+python:
+  - "2.7"
+
+install:
+   - pip install fabric 
+
+script:
+
+   - if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" = false ]; then
+       export REPO=codeforpoznan/volontulo;
```

Comment:
what's the reason of these 2 `export` lines?

Reply:
Argh! Its just commit without save before... I'll push the right version in a moment.